interface.html

```diff
@@ -390,6 +390,15 @@ <h4 class="panel-title collapsed" role="button" data-toggle="collapse" data-pare
                     <input type="text" name="fl-push-serverKey" class="form-control" id="fl-push-serverKey" />
                   </div>
                 </div>
+                <div class="form-group clearfix">
+                  <div class="col-sm-4 control-label">
+                    <label for="fl-push-projectId">Project ID</label>
+                    <p class="help-block label-helper"><small>This is a unique identifier for your Firebase project that can be found in the URL: https://console.firebase.google.com/project/<b>&lt;projectId&gt;</b> or in the project general settings in Firebase.</small></p>
```

Comment:
I've taken this sentence from here:

<img width="892" alt="screen shot 2018-08-22 at 14 31 57" src="https://user-images.githubusercontent.com/574210/44463482-27165100-a618-11e8-8995-39fca0dcac9c.png">